proposals/infra/INF-0003-packoffset.md

```diff
@@ -0,0 +1,171 @@
+<!-- {% raw %} -->
+
+# Strict Availability Diagnostics
```

Comment:
Looks like you forgot to update the title

Reply:
Fixed .